### DISCUSSION TITLE
Busco equipo para hackathon?

### DISCUSSION THREAD

**Discussion by dev-josegarcia** [Launch-X-Latam/MisionFrontEnd#586] [Team Posts]
mi correo es lx15751@innovaccion.mx , o se puede hacer un equipo.

**Comment by NickArregoces** [comment]
Hola yo tambien mi correo es lx142456@innovaccion.mx

**Comment by LuisSerranoGtz** [comment]
Hola, si hacen falta mas mi correo es lx02079@innovaccion.mx

**Comment by JuanYanez292** [comment]
Buenas, tambien busco equipo mi correo eso lx15231@innovaccion.mx

**Comment by DavidM444** [comment]
Tambien busco grupo lx07267@innovaccion.mx

**Comment by Andersonm22** [comment]
yo aun no tengo equipo, mi correo es lx10407@innovaccion.mx

**Comment by roramirez28** [comment]
Me apunto! lx02087@innovaccion.mx

**Comment by joseandresgavilanes** [comment]
Busco equipo  ***@***.***

**Comment by IsmaelMorales16** [comment]
Buenas, yo busco equipo mi correo es: Ix10470@innovaccion.mx

**Comment by gapman20** [comment]
Buenas, busco equipo aqui dejo mi correo lx16569@innovaccion.mx

**Comment by RicardoLunaRivera** [comment]
aun me puedo integrar?
dejo mi correo: lx04128@innovaccion.mx
